lib/ASRouterTargeting.jsm

```diff
@@ -67,6 +67,9 @@ const TargetingGetters = {
       update: settings.update
     };
   },
+  get currentTimestamp() {
+    return new Date(new Date().getTime());
```

Comment:
Why do we need to nest these? It looks like just doing `new Date()` returns the same as this, and when i subbed it out in the jexl expression it seems to still return the same result, and it passes the tests too

Reply:
I wanted to return a date object not a timestamp.

Reply:
```js
new Date()
Date 2018-08-25T23:02:00.134Z
```